content/post/0-to-60-analytics-stack.md

```diff
@@ -13,36 +13,36 @@ Congratulations, you are launching an analytics/data function! Maybe you are the
 
-I am going to sort the article by tool, since realistically you are making discrete decisions per tool. But when talking to the org, you need to do the opposite! It's time to put on your salesperson hat, in two specific ways:
+I am going to sort the article by tool, since that is how you are making discrete decisions. But when talking to the org, you need to do the opposite! It is time to put on your salesperson hat, in two specific ways:
 
 
-1. You are not buying tools - You are solving problems! Create your "elevator pitch", framed in terms of the benefits - faster results because X, more revenue because Y - and not in terms of "this one is more expensive, but you can visualize as a directed acyclic graph!". If two of these tools are most easily explained as solving a single problem, pitch them that way, and combine the cost estimates. Put simply, no one cares if the float cup, flapper, and flush valve cost $3, $5, and $8, they just want a working toilet and are willing to pay $20. 
+1. You are not buying tools - You are solving problems! Create your "elevator pitch", framed in terms of the benefits - faster results because X, Y more revenue because Z - and not in terms of "this tool is more expensive, but you can visualize as a directed acyclic graph!". If two of these tools are most easily explained as solving a single problem, pitch them that way, and combine the cost estimates. Put simply, no one cares if the flapper, float cup and flush valve cost $3, $5, and $8, they want a working toilet and are willing to pay $20. 
 
 
 2. You need to understand who in leadership has decision-making power and engage them early to understand the steps towards approval. Otherwise, you can waste a lot of time trotting out presentations to anyone willing to listen. 
 
-Now, I will cover some aspects of the analytics stack on a per-tool basis:
 
 
-## ETL Tool:
+## EL Tool:
 
 - Some solutions you might consider: Stitch, Fivetran, tray.io
 
-For finance, most data-movers costs far less than a full-time Data Engineer, so buying this component saves the company money from paying an engineer to do (unsatisfying) data work. However, some providers bill per number of rows synced, while some bill per number of unique data sources. You may find that the finance team prefers the billing structure of your second choice much more than your first choice provider! Be prepared to either convince finance that you can keep variable costs under control, or convince the sales rep to customize your contract.
+The data-moving component will transport data from your source (e.g. production database) to a destination where you can easily analyze it. You main consideration is to ensure your tool of choice will support any data sources you will need over the next several years.  It is worth opting for a tool that covers software you are planning on using next year to avoid the switching cost down the road, and this is easy to explain. 
```

Comment:
```suggestion
The data-moving component will transport data from your source (e.g. production database) to a destination where you can easily analyze it. You main consideration is to ensure your tool of choice will support any data sources you will need over the next several years.  It is worth opting for a tool that covers software you are planning on using next year to avoid the switching cost down the road, and this is easy to explain even to non-technical audiences. 
```